scripts/lib/run_hooks.py

```diff
@@ -30,6 +30,10 @@
 
 if args.kind == "post-deploy":
     old_dir_name = "last"
+    if not os.path.exists(DEPLOYMENTS_DIR + "/last"):
+        # Fresh installs which are doing an OS upgrade don't have a
+        # "last" yet
+        sys.exit(0)
```

Comment:
For this scenario, would it be better to pass the same version as both old and new?